Plugins/Geant4/src/Geant4Converters.cpp

```diff
@@ -301,12 +303,14 @@ std::shared_ptr<Acts::HomogeneousSurfaceMaterial>
 Acts::Geant4MaterialConverter::surfaceMaterial(const G4Material& g4Material,
                                                ActsScalar original,
                                                ActsScalar compressed) {
+  assert(g4Material.GetNumberOfElements() == 1);
```

Comment:
In fact, if is a compound, one would have to do the averaging ...

Reply:
followed up on this here https://github.com/acts-project/acts/pull/1805